pennylane/fermi/fermionic.py

```diff
@@ -523,6 +632,36 @@ def to_mat(self, n_orbitals=None):
 
         return mat
 
+    def commute(self, fw, source, target):
```

Comment:
I am not so sure about having this method. Seems like that the user can do it on their side.